pom.xml

```
@@ -46,7 +46,7 @@
         <azuresdk.version>1.19.0</azuresdk.version>
         <azure-storage.version>6.1.0</azure-storage.version>
         <jackson.version>2.10.2</jackson.version>
-        <jenkins.version>2.60.3</jenkins.version>
+        <jenkins.version>2.150.3</jenkins.version>
```

I would go to 2.164.3 instead, the azure-credentials plugin is moving to that,
ref: https://github.com/jenkinsci/azure-credentials-plugin/pull/41/files#r397998325
```suggestion
        <jenkins.version>2.164.3</jenkins.version>
```